pyflakes/checker.py

```diff
@@ -1421,7 +1421,15 @@ def ignore(self, node):
         STARRED = NAMECONSTANT = NAMEDEXPR = handleChildren
 
     def SUBSCRIPT(self, node):
-        if _is_typing(node.value, 'Literal', self.scopeStack):
```

Comment:
Even though it's explained in the test, I think it'd be useful to have a comment here too explaining why it's not using `_is_typing`. Maybe in several years it will make sense to switch back.